lib/MusicBrainz/Server/Report/LowDataQualityReleases.pm

```diff
@@ -1,4 +1,4 @@
-package MusicBrainz::Server::Report::LowQualityReleases;
+package MusicBrainz::Server::Report::LowDataQualityReleases;
```

Comment:
I'd be fine with that too if @yvanzo prefers it :)

Reply:
If you want to but I didn’t find any link to this report, even in the ticket for its creation, and it’s pretty new.

Reply:
Ok, fine, no edit notes seem to link to it either, so let's change it if you want.

Somewhat relatedly, can we make it so all the entity adding URLs are `/add` instead of `/create`? :p

Reply:
Yes but those exist for so long that redirects would be needed to avoid breaking third party form seeders.